locales/base/translation.json

```diff
@@ -2219,5 +2221,6 @@
       "contactSupport": "Contact Support",
       "dismiss": "Dismiss"
     }
-  }
+  },
+  "jumpstartReclaimDescription": "Reclaiming your link incurs a network gas fee, would you like to proceed?"
```

Comment:
nit: could we create an object to group the translations under? i think you'll need to add error messages (the generic fetch + the failed to reclaim ones) and it'd be easier to understand where those errors belong to if they're grouped within a `jumpstartReclaim` key?